### PR TITLE
Add ansible-navigator smoke test job

### PR DESCRIPTION
**Pull request by relrod**
Adds a job that can run `tox -e smoke` for running `ansible-navigator` smoke tests...

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -463,3 +463,10 @@
     vars:
       container_command: podman
     nodeset: ubuntu-bionic-2vcpu
+
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:466]: This looks right, but lets move it into zuul.d/ansible-navigator-jobs.yaml

Comment (pabelanger) [reply zuul.d/jobs.yaml:466]: You can also add the job to https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L54 which attaches it to the check / gate pipelines for the navigator project.

Comment (cidrblock) [reply zuul.d/jobs.yaml]: thx paul

**Comment by pabelanger** [comment]
@cidrblock I would actually revert this until you get the job properly working. There is no need to merge to get zuul to use speculative job configuration.

File: zuul.d/ansible-navigator-jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- job:
+    name: ansible-navigator-tox-smoke
+    parent: tox
```
Comment (pabelanger) [inline zuul.d/ansible-navigator-jobs.yaml:4]: it is possible you want to use ansible-tox-py38 here, over tox

Comment (pabelanger) [reply zuul.d/ansible-navigator-jobs.yaml:4]: this will allow you to use bindep.txt files for installs

Comment (relrod) [reply zuul.d/ansible-navigator-jobs.yaml:4]: Is there anything special I have to do for that? Just switch it here and then drop a bindep.txt in the root of the other repo?

Comment (relrod) [reply zuul.d/ansible-navigator-jobs.yaml:4]: Opened #849 to change it, but actually if I'm reading right, `tox` should be enough? https://opendev.org/zuul/zuul-jobs/src/branch/master/zuul.d/python-jobs.yaml#L36

**Pull request merged**